templates/base.html

```diff
@@ -55,6 +55,8 @@
 
   <link href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/css/style.css" type="text/css" rel="stylesheet" />
 
+    <link href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/css/mystyles.css" type="text/css" rel="stylesheet" />
```

Comment:
Oops, I didn't mean to submit that with my PR. I saw the `CSS_OVERRIDE` in the documentation but for some reason it wasn't picking up my CSS. I'm assuming something was off with the paths that I tried.